playbooks/vars/defaults.yaml

```diff
@@ -94,6 +94,11 @@ octavia_enabled: true
 
 manila_enabled: false
 
+# Enable SSL for the OpenStack public endpoints
+# using self-signed certificate
+ssl_enabled: false
```

Comment:
I think we should enable this by default but first I want people to test it and see if it breaks for anyone. So please don't ask me to enable it by default now, and let's wait a bit.